configure.ac

```diff
@@ -5442,7 +5442,20 @@ if test "$CANNOT_DUMP" != "yes"; then
         CARGO_DEFAULT_FEATURES="${CARGO_DEFAULT_FEATURES}\"unexec\", "
     fi
 fi
-
+if test "$window_system" != "none"; then
+    CARGO_DEFAULT_FEATURES="${CARGO_DEFAULT_FEATURES}\"window-system\", "
+fi
+case "$window_system" in
+    x11)
+        CARGO_DEFAULT_FEATURES="${CARGO_DEFAULT_FEATURES}\"x11\", "
+    ;;
+    nextstep)
+	CARGO_DEFAULT_FEATURES="${CARGO_DEFAULT_FEATURES}\"nextstep\", "
+    ;;
+    w32)
```

Comment:
The `w32` is likely too generic. Perhaps `window_system_x`, `window_system_nextstep`, and `window_system_ms_win`?

Reply:
I was worried that would be an issue. Done, though I used dashes as that's the convention with feature names.